src/main/rx/rx.c

```diff
@@ -332,9 +332,22 @@ static void processRxChannels(void)
             rxCheckPulse(chan, sample);
         }
 
-        // validate the range
-        if (sample < PULSE_MIN || sample > PULSE_MAX)
-            sample = rxConfig->midrc;
+        // validate and fix the sample
+        if (sample < PULSE_MIN || sample < rxConfig->rx_min_usec) {
+            if (chan < 4) {
+                // stick channel
+                sample = rxConfig->midrc;
```

Comment:
quite possibly not, for 3d mode flight that would cause the motor to stop.  but existing behaviour does that anyway.
for normal flight mid throttle should be for hovering, so midrc should be safe there.